apps/emqx_resource/src/emqx_resource_buffer_worker.erl

```diff
@@ -1344,9 +1346,9 @@ is_inflight_full(InflightTID) ->
     Size >= MaxSize.
 
 inflight_count(InflightTID) ->
-    case ets:info(InflightTID, size) of
-        undefined -> 0;
-        Size -> max(0, Size - ?INFLIGHT_META_ROWS)
+    case ets:lookup(InflightTID, ?BATCH_SIZE_REF) of
+        [] -> 0;
```

Comment:
I think it would only happen in some race condition where the table was deleted but some code (possibly a late callback) calls it afterwards.  This is mostly to avoid reports about strange "crashes" when a bridge is (re)starting / shutting down.

Reply:
:heavy_check_mark:

Reply:
BTW just found out that there is a [`emqx_utils_ets:lookup_value/3`](https://github.com/emqx/emqx/blob/0c727fc9b2da945638d3db1b3dc521fb72ec3d0d/apps/emqx_utils/src/emqx_utils_ets.erl#L52-L53). 😅

Reply:
🫠 :see_no_evil: 

Thanks!  I'll use that :beers: